blog/admin.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib import admin
+from django.contrib.auth.models import Group
+
+from blog.models import Post, Commentary, User
+
+admin.site.unregister(Group)
+
+
+@admin.register(User)
+class UserAdmin(admin.ModelAdmin):
+    list_display = [
+        field.name for field in User._meta.fields
+    ]  # Lists all fields in User model
+
+
+@admin.register(Post)
+class PostAdmin(admin.ModelAdmin):
+    list_display = [
+        field.name for field in Post._meta.fields
+    ]  # Lists all fields in Post model
```

Comment:
Same as the previous comment, it's better to explicitly define the fields for `list_display`.